c7n/resources/ebs.py

```diff
@@ -399,6 +399,43 @@ class resource_type(object):
     action_registry = actions
 
 
+@EBS.action_registry.register('detach')
+class VolumeDetach(BaseAction):
+
+    """
+    Detach an EBS volume from an Instance.
+
+    If 'Force' Param is True, then we'll do a forceful detach
+    of the Volume. The default value for 'Force' is False.
+
+     :example:
+
+         .. code-block:: yaml
```

Comment:
docs generation is failing because this is indent too far out, this should be in line with `:example:` like so:

```python
     :example:

     .. code-block:: yaml
```